tests/cypress/integration/reportRepair/smellGas.spec.js

```diff
@@ -0,0 +1,11 @@
+describe('smellGas', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/report-repair/smell-gas');
+  })
+  it('Displays non-mvp title', () => {
```

Comment:
Same as my [other comment](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/pull/38/files#diff-2677b9d15a16faaef3c6937fbacb1436f30692b12d8713d7b9a19aeee6f1c855R8) - create a separate test for each thing being checked.